test/unit/lib/plugins/aws/package/compile/events/kafka.test.js

```diff
@@ -340,14 +345,13 @@ describe('test/unit/lib/plugins/aws/package/compile/events/kafka.test.js', () =>
       await runCompileEventSourceMappingTest(eventConfig);
     });
 
-    it('should correctly compile EventSourceMapping resource properties for SASL_SCRAM_512_AUTH', async () => {
+    it('should correctly compile EventSourceMapping resource properties for CLIENT_CERTIFICATE_TLS_AUTH', async () => {
```

Comment:
git diff is being a bit fussy here, but this is the test for adding the client certs access configuration